dask/array/chunk.py

```diff
@@ -67,11 +67,7 @@ def keepdims_wrapped_callable(x, axis=None, keepdims=None, *args, **kwargs):
 nansum = np.nansum
 nanprod = np.nanprod
 
-try:
-    from numpy import nancumprod, nancumsum
-except ImportError:  # pragma: no cover
-    nancumprod = npcompat.nancumprod
-    nancumsum = npcompat.nancumsum
+from numpy import nancumprod, nancumsum
```

Comment:
Neither `nancumprod` nor `nancumsum` appear to be used in this module. We may be able to drop this `import` entirely

Reply:
Nvm, while `nancumprod` and `nancumsum` aren't used in this module they do get imported from this module in `dask/array/reductions.py`

Reply:
Doing

```python
nancumprod = np.nancumprod
nancumsum = np.nancumsum
```

should work and match the surrounding imports from numpy

Reply:
Makes sense. Have pushed such a commit. Thanks for the tip.